br/pkg/storage/gcs.go

```diff
@@ -423,14 +427,14 @@ skipHandleCred:
 	}
 
 	httpClient := opts.HTTPClient
-	if reqRec := recording.GetRequests(ctx); reqRec != nil {
```

Comment:
to meter traffic, we need to have it inside object reader/writer, the old context var not fit in this scenario, so it's better pass it as a option